drivers/hud_lsa/app/controllers/hud_lsa/lsas_controller.rb

```diff
@@ -98,6 +98,17 @@ def download_intermediate
       @filter.update(filter_params) if filter_params.present?
     end
 
+    def filter_params
+      return {} unless params[:filter]
+
+      filter_p = params.require(:filter).permit(filter_class.new.known_params)
+      filter_p[:user_id] = current_user.id
+      filter_p[:enforce_one_year_range] = false
```

Comment:
Maybe
```suggestion
      filter_p = super
      return {} unless filter_p.present?
      
```
To ensure you get the all the behavior from the base class?